roles/lvm_snapshots/tasks/revert.yml

```
@@ -22,3 +22,12 @@
     path: "/root/boot-backup-{{ lvm_snapshots_set_name }}.tgz"
     state: absent
   when: lvm_snapshots_boot_backup
+
+- name: Wait for the snapshot to drain
+  ansible.builtin.command: "lvs --select 'lv_name = {{ item.origin }}' --reportformat json"
```

To guarantee  we are looking at the expected LV?... 
```suggestion
  ansible.builtin.command: "lvs --select 'vg_name = {{ item.vg_name }} && lv_name = {{ item.origin }}' --reportformat json"
```